apps/web/modules/event-types/views/event-types-listing-view.tsx

```diff
@@ -171,6 +172,13 @@ const Item = ({
   const parsedeventTypeColor = parseEventTypeColor(type.eventTypeColor);
   const eventTypeColor =
     parsedeventTypeColor && parsedeventTypeColor[hasDarkTheme ? "darkEventTypeColor" : "lightEventTypeColor"];
+  const totalBookings = trpc.viewer.bookings.get.useQuery({
+    filters: {
+      eventTypeIds: [type.id],
+      status: "past",
+    },
+    limit: BOOKING_LIMIT,
+  }).data?.bookings.length;
```

Comment:
@PeerRich the total booking badge would be useless for orgs and bug team. as would have more than 50+ bookings for most of the event types

Reply:
Plus this would be really slow query for them

Reply:
@Udit-takkar any ideas how to fix this?